packages/next-auth/src/next/index.ts

```diff
@@ -113,6 +113,11 @@ function NextAuth(
   res: NextApiResponse,
   options: AuthOptions
 ): any
+function NextAuth(
+  req: NextRequest,
+  context: any,
```

Comment:
We could possibly type `context` here as [RouteModuleHandleContext](https://github.com/vercel/next.js/blob/9bde7dcc0f8c73f5785ec6e0efd8e59e7a7c4c4b/packages/next/src/server/future/route-modules/route-module.ts#L20C18-L20C42) but the type export location will probably change in future nextjs releases.